### PR TITLE
planner/core: refresh stale regions in cache for batch cop response

### PR DESCRIPTION
**Pull request by hanfei1991**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tics/issues/1873

### What is changed and how it works?

What's Changed:
add retry regions in Batch cop response. then refresh the stale regions in cache.

### Related changes
https://github.com/pingcap/kvproto/pull/761

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

### Release note 

- planner/core: refresh stale regions in cache for batch cop response

### CODE REVIEW

File: store/copr/batch_coprocessor.go [hunk 1]
```diff
@@ -410,6 +410,15 @@ func (b *batchCopIterator) handleBatchCopResponse(bo *tikv.Backoffer, response *
 		return errors.Trace(err)
 	}
 
+	if len(response.RetryRegions) > 0 {
+		for _, retry := range response.RetryRegions {
+			id := tikv.NewRegionVerID(retry.Id, retry.RegionEpoch.ConfVer, retry.RegionEpoch.Version)
+			logutil.BgLogger().Info("invalid region because tiflash detected stale region", zap.String("region id", id.String()))
```
Comment (lzmhhh123) [inline store/copr/batch_coprocessor.go:416]: Will this statement print many logs when a tiflash peer is down?

Comment (lzmhhh123) [reply store/copr/batch_coprocessor.go:416]: How about print the `len(retryRegions)` and three of them.

**Review by lzmhhh123 — COMMENTED** [review]
Rest LGTM

File: store/copr/batch_coprocessor.go (hunk 1 above)
Comment (hanfei1991) [reply store/copr/batch_coprocessor.go:417]: fixed, only print the first ten ones.

**Review by lzmhhh123 — APPROVED** [review]

**Pull request merged**